manifest.appcache

```diff
@@ -1,3 +1,6 @@
+# Random number: 23
+# Random number (or any change) necessary to update client cache.
```

Comment:
Usually, explanatory comments are before the item they are explaining.

This makes it look like this was another line that's necessary to accomplish updating the client cache, where the above line is enough.

You could even merge the two lines. Something like `Version (increment every time to update the cache for clients): 1`.